src/stores/oidc/OidcClientStore.ts

```diff
@@ -29,17 +29,21 @@ import { getDelegatedAuthAccountUrl } from "../../utils/oidc/getDelegatedAuthAcc
 export class OidcClientStore {
     private oidcClient?: OidcClient;
     private initialisingOidcClientPromise: Promise<void> | undefined;
-    private authenticatedIssuer?: string;
+    private authenticatedIssuer?: string; // set only in OIDC-native mode
     private _accountManagementEndpoint?: string;
+    public readonly readyPromise: Promise<void>;
```

Comment:
document please. What does this promise mean? (or rather, what does "ready" mean in this context?)

Can it fail?